packages/react-components/src/components/assets/RaisedHandIcon.tsx

```diff
@@ -0,0 +1,835 @@
+// Copyright (c) Microsoft Corporation.
```

Comment:
Hmm this is a pretty big svg.. gzip size of this is 5KB cc @PorterNan if this is OK or should we find an alternative solution

Reply:
Found this one to compress the svg a bit, from 47k to 13k - https://www.iloveimg.com/compress-image/compress-svg
Not sure how it works, but if that could make the size smaller then let's do it

13k is still a bit too huge for one icon, but at least a lot better than 50k